src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -87,14 +84,6 @@ public TreeWalker() {
         setFileExtensions("java");
     }
 
-    /**
-     * Sets classLoader to load class.
-     * @param classLoader class loader to resolve classes with.
-     */
-    public void setClassLoader(ClassLoader classLoader) {
```

Comment:
if we keep such method in Checker, it is better to keep it in TreeWalker (with empty body).

Reply:
Will try, but it might be the same as https://github.com/checkstyle/checkstyle/pull/7114#issuecomment-536146623

Reply:
Done.